Source/WebCore/inspector/WorkerInspectorController.cpp

```diff
@@ -99,11 +99,33 @@ void WorkerInspectorController::workerTerminating()
     m_debugger = nullptr;
 }
 
-void WorkerInspectorController::connectFrontend()
+void WorkerInspectorController::frontendInitialized()
+{
+#if ENABLE(REMOTE_INSPECTOR)
+    if (m_pauseAfterInitialization) {
+        m_pauseAfterInitialization = false;
+
+        ensureDebuggerAgent();
+        m_debuggerAgent->enable();
+        m_debuggerAgent->pause();
```

Comment:
i think this would be better as
```cpp
ensureDebuggerAgent().enable();
ensureDebuggerAgent().pause();
```
to match other cases and just in case `WorkerDebuggerAgent::enable` somehow causes `m_debuggerAgent` to be cleared